src/main/java/org/datavyu/util/ClockTimer.java

```diff
@@ -142,6 +142,15 @@ public synchronized double getStreamTime() {
         return (long) clockTime + minTime;
     }
 
+    /**
+     * Get the current Clock time
```

Comment:
Please add a note that this is the current time within the tolerance of the periodic sync

Reply:
OK.